save-backend/src/main/kotlin/com/saveourtool/save/backend/service/LnkOrganizationTestSuiteService.kt

```diff
@@ -0,0 +1,79 @@
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.backend.service
+
+import com.saveourtool.save.backend.repository.LnkOrganizationTestSuiteRepository
+import com.saveourtool.save.domain.Role
+import com.saveourtool.save.entities.*
+import com.saveourtool.save.permission.Rights
+import com.saveourtool.save.utils.orNotFound
+import org.springframework.stereotype.Service
+
+/**
+ * Service of [LnkOrganizationTestSuite]
+ */
+@Service
+class LnkOrganizationTestSuiteService(
+    private val lnkOrganizationTestSuiteRepository: LnkOrganizationTestSuiteRepository,
+) {
+    /**
+     * @param organization
+     * @return all [TestSuite]s with rights for [organization]
+     */
+    fun getAllTestSuitesByOrganization(organization: Organization) =
+            lnkOrganizationTestSuiteRepository.findByOrganization(organization)
+                .map {
+                    it.testSuite
+                }
+
+    /**
+     * @param organization
+     * @param testSuite
+     * @return [LnkOrganizationTestSuite] by [organization] and [testSuite]
+     */
+    fun findByOrganizationAndTestSuite(organization: Organization, testSuite: TestSuite) =
+            lnkOrganizationTestSuiteRepository.findByOrganizationAndTestSuite(organization, testSuite)
+
+    /**
+     * Set [rights] of [organization] over [testSuite].
+     *
+     * @throws IllegalStateException if [rights] is [Role.NONE]
+     */
+    @Suppress("KDOC_WITHOUT_PARAM_TAG", "UnsafeCallOnNullableType")
```

Comment:
where is `UnsafeCallOnNullableType`, and maybe add `@return` in kdoc?

Reply:
Will remove `UnsafeCallOnNullableType`
But it doesn't return anything...

Reply:
Done